charts/castai-spot-handler/Chart.yaml

```
@@ -2,5 +2,5 @@ apiVersion: v2
 name: castai-spot-handler
 description: CAST AI spot handler daemonset chart
 type: application
-version: 0.8.0
+version: 0.9.0
```

```suggestion
version: 0.10.0
```

We have https://github.com/castai/helm-charts/releases/tag/castai-spot-handler-0.9.0